src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx

```diff
@@ -200,7 +200,31 @@ const BaseConfiguration = () => {
               })}
             />
           </div>
-          <div className="right"></div>
+          <div className="right">
+            <RadioButton
+              row
+              id="patient-basis"
+              dataTestId="patient-basis"
+              label="Patient Basis:"
```

Comment:
I think no need of colons here

Reply:
ooh yeah you are right, I will remove them